src/commands/lambda/constants.ts

```diff
@@ -1,46 +1,51 @@
 export const DD_LAMBDA_EXTENSION_LAYER_NAME = 'Datadog-Extension'
 export const EXTENSION_LAYER_KEY = 'extension'
+export const DOTNET_RUNTIME = 'dotnetcore3.1'
 export const RUNTIME_LAYER_LOOKUP = {
```

Comment:
I know it was already there, but I'm not sure `RUNTIME_LAYER_LOOKUP` is the best name for that map. To me that would suggest that we can use this to get the layer for a specific runtime. That is mostly what this does, but it also includes the extension which is not a runtime per se. Maybe a better name would just be `LAYER_LOOKUP` or something like that.